python/sgp4_scanner_box_plot.py

```diff
@@ -0,0 +1,213 @@
+  if errorType[ errorTypeIndex ] == "arrival_position":
+    yLabel = 'Position error [km]'
+    figureNameLabel = 'position error'
+    annotation_xOffset = 0.18
+    annotation_yOffset = 0
+    print "Plotting position error box plot ..."
+    print ""
+    leo_magnitudeError = np.array( leo_scan_data[ 'positionErrorMagnitude' ], dtype=float )
+    heo_magnitudeError = np.array( heo_scan_data[ 'positionErrorMagnitude' ], dtype=float )
+    geo_magnitudeError = np.array( geo_scan_data[ 'positionErrorMagnitude' ], dtype=float )
+    magnitudeError = []
+    magnitudeError.append( leo_magnitudeError )
+    magnitudeError.append( heo_magnitudeError )
+    magnitudeError.append( geo_magnitudeError )
+  else:
+    yLabel = 'Velocity error [km/s]'
+    figureNameLabel = 'velocity error'
+    annotation_xOffset = 0.12
+    annotation_yOffset = 0.003
+    print ""
+    print "Plotting velocity error box plot ..."
+    print ""
+    leo_magnitudeError = np.array( leo_scan_data[ 'velocityErrorMagnitude' ], dtype=float )
+    heo_magnitudeError = np.array( heo_scan_data[ 'velocityErrorMagnitude' ], dtype=float )
+    geo_magnitudeError = np.array( geo_scan_data[ 'velocityErrorMagnitude' ], dtype=float )
+    magnitudeError = []
+    magnitudeError.append( leo_magnitudeError )
+    magnitudeError.append( heo_magnitudeError )
+    magnitudeError.append( geo_magnitudeError )
+
+  fig = plt.figure( )
+  axes = fig.gca( )
+  # plotOut = plt.boxplot( magnitudeError, showfliers=False )
+  plotOut = plt.boxplot( magnitudeError, sym='' )
+  plt.setp( plotOut['boxes'], color='black' )
+  plt.setp( plotOut['whiskers'], color='black' )
+  plt.setp( plotOut['medians'], color='black' )
+  plt.setp( plotOut['fliers'], color='black' )
+  plt.xticks( [1, 2, 3], ['SSO', 'GTO', 'GEO'] )
+  plt.xlabel( 'Orbit type' )
+  plt.ylabel( yLabel )
+
+  for line in plotOut['medians']:
+    # Get position data for the median line (from matplotlib.lines)
+    medianX, medianY = line.get_xydata( )[ 1 ]
+
+    # Overlay median values
+    axes.annotate( '%1.2f' % medianY,                                                             \
+                   xy=( medianX + annotation_xOffset, medianY + annotation_yOffset ),             \
+                   xycoords='data',                                                               \
+                   horizontalalignment='center',                                                  \
+                   verticalalignment='center' )
+
+  # Save figure to file.
+  plt.savefig( output_path_prefix + config['figure_name'] + ' ' + figureNameLabel
+               + config["figure_format"], dpi=300 )
+  plt.close( )
+
+print "All figures generated successfully!"
+print ""
+
+# Close SQLite databases if they're still open.
+if leo_database:
+  leo_database.close( )
+
+if heo_database:
+  heo_database.close( )
+
+if geo_database:
+  geo_database.close( )
+
+# Stop timer
+end_time = time.time( )
+
+# Print elapsed time
```

Comment:
Add a dashed line before script time to distinctly distinguish it from the output of the calculations.